crates/codspeed/src/codspeed.rs

```diff
@@ -19,7 +19,8 @@ pub struct CodSpeed {
 
 impl CodSpeed {
     pub fn new() -> Self {
-        let is_instrumented = measurement::is_instrumented();
+        let ih = InstrumentHooks::instance();
```

Comment:
not a fan of abbreviations overall, but here it does not bring that much, how about `hooks` ? or `hooks_instance` or `instrument_hooks_instance` ?